modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -29,7 +29,7 @@
   >
     <slot name="before" />
     <component
-      v-for="field in schema" :key="field.name"
+      v-for="field in schema" :key="field.name.concat(field._id)"
```

Comment:
```suggestion
      v-for="field in schema" :key="field.name.concat(field._id ?? '')"
```
Just to be on the safe side, could you please update with safe guard.